src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <h1 class="visually-hidden">Search</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="get"
+    role="search"
+    class="search"
+    data-qa="big"
+    >
+      <div class="search__bar search__bar--big">
+        <input
+     aria-label="search"
+     type="search"
+     placeholder="Try &quot;Los Angeles&quot;"
+     class="search__input search__input--big"
+     data-qa="keypress"
+        >
+      </div>
+    </form>
```

Comment:
[read](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) code styling recommendations and apply all over the page similar to these ones, please
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="get"
      role="search"
      class="search"
      data-qa="big"
    >
      <div class="search__bar search__bar--big">
        <input
          aria-label="search"
          type="search"
          placeholder="Try &quot;Los Angeles&quot;"
          class="search__input search__input--big"
          data-qa="keypress"
        >
      </div>
    </form>
```

Reply:
is not fixed